pyproject.toml

```diff
@@ -36,9 +36,8 @@ dependencies = [
    "jsonschema",
    "requests",
    "tqdm",
-   "datasets",
-   "pycountry",
-   "pyairports",
```

Comment:
`pyairports` is necessary for `import outlines`. It's wrapped in a try-except block, but the block doesn't pass.

Reply:
I see. then these try-except blocks can be removed, right? Since it will now never enter the `except` part; in order for people to be able to run that code, they have to install `outlines`, which has `pyairports` and `pycountry` as a direct dependency.

I added both `pyairports` and `pycountry` back to the dependencies. Let me know if you want me to make any further changes regarding this, e.g. remove the `try-except` block.